microbench/src/main/java/io/netty/buffer/ByteBufAccessBenchmark.java

```diff
@@ -149,6 +149,14 @@ public ByteBuf setLong() {
     public int readBatch() {
         buffer.readerIndex(0).touch();
         int result = 0;
+        //WARNING!
```

Comment:
nit: can you add a space after `//` in each line ;)

Reply:
Done!